### PR TITLE
Add OpenAPI Specification to Release Assets

### PR DESCRIPTION
**Pull request by kiblik**
In the future, I would like to add API changes to release notes. As preparation, I would like to release OpenAPI Specifications with each release as assets.

### CODE REVIEW

**Comment by mtesauro** [comment]
@kiblik FYI: I've been working on this from the other sided - aka creating way to diff spec files from DefectDojo's API. That's about 80% done and will show changes / breaking changes between any 2 versions of the API spec files.

**Comment by mtesauro** [comment]
I'm using this library - https://github.com/Tufin/oasdiff and am aiming towards a GH action which runs the tools and produces the needed output

**Comment by kiblik** [comment]
> I'm using this library - https://github.com/Tufin/oasdiff and am aiming towards a GH action which runs the tools and produces the needed output

I was planning to use the exact tool :)
But you still need some base. How did you plan to generate the previous version OpenAPI definition?
My plan was to store it as part of a release and just fetch it during the following release.

**Comment by mtesauro** [comment]
@kiblik Is this ready to review/approve?

**Comment by kiblik** [comment]
> @kiblik Is this ready to review/approve?

Yes, it is :)

**Review by Maffooch — APPROVED** [review]

**Review by devGregA — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -16,11 +16,48 @@ on:
...
+        id: create_release
+        uses: release-drafter/release-drafter@v5.25.0
         with:
           version: ${{github.event.inputs.version}}  
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Upload Release Asset - OpenAPI Specification - YAML
+        id: upload-release-asset-yaml
+        uses: sekwah41/upload-release-assets@v1
```
Comment (cneill) [inline .github/workflows/release-drafter.yml:45]: Is there a reason to use this fork over `actions/upload-release-asset`? I know the official one is archived but I'm a little hesitant to use a random fork here..

Comment (kiblik) [reply .github/workflows/release-drafter.yml:45]: > Is there a reason to use this fork over `actions/upload-release-asset`? I know the official one is archived but I'm a little hesitant to use a random fork here..

As I remember, there is no other reason (only stopped maintenance).
If you prefer, I can run a local test with the original official one.

Comment (kiblik) [reply .github/workflows/release-drafter.yml]: Not sure if I'm doing sth wrong but I'm not able to make it work with `actions/upload-release-assets`
- https://github.com/kiblik/django-DefectDojo/actions/runs/7188225312/job/19577606201
- https://github.com/kiblik/django-DefectDojo/actions/runs/7188488164/job/19578368890

Any help is welcome

**Review by mtesauro — APPROVED** [review]
Approved

**Pull request merged**